### PR TITLE
Fix golint errors in controller replication and replica_set packages

### PR DESCRIPTION
**Pull request by ghost**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix golint issues in `pkg/controller/replication`
Fix golint issues in `pkg/controller/replicat_set`

**Which issue(s) this PR fixes**:

Ref https://github.com/kubernetes/kubernetes/issues/68026

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/replication/replication_controller.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // ### ATTENTION ###
 //
-// ReplicationManager is now just a wrapper around ReplicaSetController,
+// Manager is now just a wrapper around ReplicaSetController,
```
Comment (mattjmcnaughton) [inline pkg/controller/replication/replication_controller.go]: Nit: Think we want to update `ReplicaSetController` to just be `Controller`.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/ok-to-test

Agree w/ avoiding the stutter here :)

**Review by johnSchnake — APPROVED** [review]

File: pkg/controller/replicaset/replica_set.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // For RC, the objects are converted on the way in and out (see ../replication/),
 // as if ReplicationController were just an older API version of ReplicaSet.
 // However, RC and RS still have separate storage and separate instantiations
-// of the ReplicaSetController object.
+// of the Controller object.
```
Comment (soltysh) [inline pkg/controller/replicaset/replica_set.go]: I'm not a big fan of blindly following golint errors, I'd prefer we don't change these.

File: pkg/controller/replication/replication_controller.go (hunk 1 above)
Comment (soltysh) [inline pkg/controller/replication/replication_controller.go]: I'm not a big fan of blindly following golint errors, I'd prefer we don't change these.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**